cmake/Tools/Platform/Android/android_support.py

```diff
@@ -1573,22 +1574,22 @@ class InstalledPackage(object):
         def __init__(self, installed_package_components):
             assert len(installed_package_components) == 4, '4 sections expected for installed package components (path, version, description, location)'
             self.path = installed_package_components[0]
-            self.version = LooseVersion(installed_package_components[1])
+            self.version = Version(installed_package_components[1].strip().replace(' ', '.'))  # Fix for versions that have spaces between the version number and potential non-numeric versioning (PEP-0440)
```

Comment:
nit: This operation is repeated a few times with a clarification comment, it might be worth moving it to a function.